gf/test/three_index_gf_test.cpp

```diff
@@ -385,6 +384,7 @@ TEST_F(ThreeIndexGFTest, tailPrint)
 TEST_F(ThreeIndexGFTest, DefaultConstructive)
 {
     gf_type gf_empty;
+#ifndef NDEBUG
```

Comment:
The same comment as https://github.com/ALPSCore/ALPSCore/pull/408/files#diff-f5308a091bc5c9a3c0d44b94de128649R256